build.gradle.kts

```diff
@@ -32,17 +33,17 @@ repositories {
 
 dependencies {
 
-    val springdocVersion = "1.6.13"
+    val springdocVersion = "1.6.15"
```

Comment:
Må oppgradere springdoc til 2.2.0 for at swagger skal funke med spring boot 3. Litt endringer i avhengigheter

 <dependency>
            <groupId>org.springdoc</groupId>
            <artifactId>springdoc-openapi-starter-common</artifactId>
            <version>${springdoc.version}</version>
        </dependency>
        <dependency>
            <groupId>org.springdoc</groupId>
            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
            <version>${springdoc.version}</version>
        </dependency>